cpp/ql/test/experimental/query-tests/Security/CWE/CWE-415/semmle/tests/DoubleFree.expected

```diff
@@ -0,0 +1,2 @@
+| test.c:9:8:9:10 | buf | This pointer may be cleared again later. |
```

Comment:
I think the warning should be on line 10. Since this is where the double free happens.